torch_em/data/raw_image_collection_dataset.py

```diff
@@ -81,11 +81,9 @@ def _sample_bounding_box(self, shape):
     def _ensure_patch_shape(self, raw, have_raw_channels):
         shape = raw.shape
         if any(sh < psh for sh, psh in zip(shape, self.patch_shape)):
```

Comment:
We need to also do this here: (see the corresponding function in `image_collection_dataset`):
```
if have_raw_channels and channel_first:
    shape = shape[1:]
```
and hence we also need to pass `channel_first` as an argument to this function.